apps/omg_watcher_rpc/lib/web/validators/order.ex

```diff
@@ -58,8 +58,8 @@ defmodule OMG.WatcherRPC.Web.Validator.Order do
   end
 
   defp parse_fee(map) when is_map(map) do
```

Comment:
hey @mederic-p in line https://github.com/omisego/elixir-omg/pull/1260/files#diff-68ed50a7de191c4eb70067419fd0b717R44
there are outstanding module aliases and macros, could you move them at the top?